dev/elie_wanko/modules/lg.py

```diff
@@ -0,0 +1,41 @@
+# Imports
+from numpy import random, array
+from math import floor
+
+from sklearn.linear_model import LogisticRegression
+
+def __init__():
+    pass
+    
+def classfier(penalty='l2', dual=False, tol=0.0001, C=1.0, fit_intercept=True, intercept_scaling=1, class_weight=None, random_state=None, solver='liblinear', max_iter=100, multi_class='auto', verbose=0, warm_start=False, n_jobs=None, l1_ratio=None, *, subsets, data):
+    
+    """ Returns the predicted values computed from a simple majority vote of the nearest neighbors of each point.
```

Comment:
This needs to be updated.

Reply:
Yeah. Will do, I made some changes after sending the pull request. I will add some comments as to the effects of different parameter choices on model performance.